### DISCUSSION TITLE
Elite Dangerous: Turning off bounty messages...?

### DISCUSSION THREAD

**Discussion by Ravenloff** [EDCD/EDDI#2203] [EDDI VoiceAttack Q&A]
Just started messing with Odyssey's on-foot conflict zones. Having the EDDI plugin chirp up every time I kill someone (and it happens a lot in FPS zones) it pretty annoying. 

Likewise, I went full-on pirate and illegally attacked a settlement in my SRV. Every couple of seconds I would get a new bounty on me, which the plugin dutifully mentioned. The problem is that when you're doing this sort of illegal action, those stack up quickly. There were so many that it was still reading off bounties when I had already left the settlement and was 5k away.

I went into EDDI's config, created a second personality and disable all the bounty options, but I'm still getting those messages in-game. How do I turn them off completely without affecting anything else?

**Comment by Tkael** [comment]
Do you mean that the speech responder is still speaking the messages? If you've disabled the scripts in your current selected personality then that shouldn't be happening.

Or do you mean that VoiceAttack is still triggering commands?

I need a little more clarity about what you are observing.
